### PR TITLE
Update docker images

### PR DESCRIPTION
**Pull request by kokosing**
Update docker images

### CODE REVIEW

File: .mvn/modernizer/violations.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0"?>
```
Comment (Praveen2112) [inline .mvn/modernizer/violations.xml]: If we refer it from airbase do we still need it ?

File: .travis.yml [hunk 2]
```diff
@@ -9,11 +9,11 @@ git:
   quiet: true
 
 install:
-  - mvn install -B -V -DskipTests -Dair.check.skip-all
-  - mvn dependency:go-offline -B -Dsilent
+  - ./mvnw install -B -V -DskipTests -Dair.check.skip-all
```
Comment (Praveen2112) [inline .travis.yml:12]: I guess there was a PR which migrates it to GHA

**Review by Praveen2112 — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (kokosing) [reply .travis.yml:12]: I know. It will need to be rebased.

File: .mvn/modernizer/violations.xml (hunk 1 above)
Comment (kokosing) [reply .mvn/modernizer/violations.xml]: removed

**Pull request merged**